### ISSUE TITLE
Request for pop dark gnome shell: darker background of selected apps in the search of the gnome dash

### ISSUE THREAD

**Issue by Feichtmeier** [system76/pop-gtk-theme#27]
**NOTE: Before reporting a bug, please check whether it also occurs with other themes.**
> If the bug also occurs with other themes (especially upstream default theme),
it is very likely an upstream issue.

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
**Reporter info**

```
 * Distribution - ubuntu 17.10
 * Gtk+ 3.0 version - 3.22.19-0ubuntu1
 * Desktop environment - GNOME
 * Pop version - 2.1.1.6
 ```

**Related Application and/or Shell theming (or name of widget)**
GNOME Shell 

**Actual issue**
When you search for apps background of the seleced app is very hard to see in the pop-dark/compact gnome shell theme, see the screenshot or try it yourself :) I think just a little bit less transparence could do the trick

![bildschirmfoto von 2017-09-07 23-58-17](https://user-images.githubusercontent.com/15329494/30187220-02f54e86-9429-11e7-954a-67926973970f.png)

**Comment by isantop** [comment]
I think these should probably just be the same color between the light and dark variants. They aren't light enough to really darken up the dark theme, and there really isn't any need to change it per-variant, given that the background overlay color doesn't change (And you're inherently going to have contrast issues when you put a dark overlay on top of a dark overlay).

**Comment by isantop** [comment]
Fixed in d9145d6
